project_move_activity_tasks/wizard/project_move_activity_tasks_wizard.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="view_move_activity_tasks_wizard" model="ir.ui.view">
+        <field name="name">Move activity tasks Wizard</field>
+        <field name="model">move.activity.tasks.wiz</field>
+        <field name="arch" type="xml">
+            <form string="Move activity tasks Wizard">
+                <div>
+                    <field name="message" readonly="1"/>
+                </div>
+                <group>
+                    <field name="moving_type" widget="radio"/>
+                </group>
+                <group align="left" attrs="{'invisible': [('moving_type','=', 'date')]}">
+                    <field name="days" string="Days"/>
+                    <field name="hours" string="Hours"/>
+                    <field name="minutes" string="Minutes"/>
+                </group>
+                <group attrs="{'invisible': [('moving_type','=', 'date')]}">
+                    <field name="moving_direction" widget="radio"/>
+                </group>
+                <group align="right" attrs="{'invisible': [('moving_type','=', 'interval')]}">
+                    <field name="moving_date" string="Moving date"/>
+                </group>
+                <notebook>
+                        <page string="Tasks">
+                            <field name="child_ids" string="Task" context="{
+                            'form_view_ref':'project_event.project_event_task_form',
```

Comment:
I think the problem is: wrong view, it should be  a tree view. not a project_event.project_event_task_form